samples/cpp/speech_sample/main.cpp

```diff
@@ -49,7 +49,10 @@ int main(int argc, char* argv[]) {
         BaseFile* fileOutput;
         ArkFile arkFile;
         NumpyFile numpyFile;
-        auto extInputFile = fileExt(FLAGS_i);
+        std::pair<std::string, std::vector<std::string>> input_data;
```

Comment:
Why just pair is used? Can we have different filenames for different inputs?

Reply:
Because in this function we split the data into a list of names and a list of files passed in sequential order. So we don't need structure like as key:value

Reply:
Actually I still don't understand why you use such structure. Why not ordered map or vector of pairs?

Filesnames are concatenated, but names is still in vector. Looks strange and unaligned. How different filenames correlates with different input names?

Please add apropriate test for model with 2 inputs or outputs.

Reply:
We have tests with 2 inputs/outputs, for example: test_data_iname.
I updated using of new delimeter in these tests